ApplyToBecomeInternal/ApplyToBecome.Data/Services/AcademyConversionProjectRepository.cs

```diff
@@ -22,9 +23,11 @@ public AcademyConversionProjectRepository(IHttpClientFactory httpClientFactory)
 			_httpClient = httpClientFactory.CreateClient("TramsClient");
 		}
 
-		public async Task<ApiResponse<ApiV2Wrapper<IEnumerable<AcademyConversionProject>>>> GetAllProjects(int page = 1, int count = 50, string statusFilters = "")
+		public async Task<ApiResponse<ApiV2Wrapper<IEnumerable<AcademyConversionProject>>>> GetAllProjects(int page, int count, string statusFilters = "",
+			string titleFilter = null)
```

Comment:
Why is titleFilter null and statusFilter string.empty

Reply:
Good point - probably because titleFilter was added via Resharper's 'Change Signature' refactoring.